packages/DApp/src/components/top/TopBar.tsx

```diff
@@ -55,15 +57,31 @@ export function TopBar() {
                   e.stopPropagation()
                   setIsOpened(!isOpened)
                 }}
+                isActive={isActive}
               >
+                {!isActive && '⚠️ '}
```

Comment:
I was thinking about it as well, that totally makes sense, but would it be enough for you? Let's change it to yellow / red circle and let's leave warning sign in the subnav?

Reply:
it's really non-blocking, but yes. however, in the subnav I'd leave just a text, which btw I've commented on vercel amongst some other points

Reply:
> in the subnav I'd leave just a text, which btw I've commented about its styling on vercel amongst some other points
> – https://vercel.live/link/community-dapp-git-jkbktl-networks-status-im-web.vercel.app?page=%2Fvotes%3FvercelThreadId%3D5g3qZ&via=in-app-timestamp

What about this? Worth addressing now as well, or would you prefer not to?

Reply:
@felicio I consider it fixed as it's now better positioned than before, otherwise there's no design for similar element, moreover this is truly marginal thing as user is prompted immediately after wallet connection, it should be visible very sporadically just for user who consciously reject switching network